dev-client/src/components/home/BottomSheet.tsx

```diff
@@ -63,6 +64,24 @@ const EmptySiteMessage = () => {
   );
 };
 
+/**
+ * This function calculated the initial snap value to the bottom sheet depending on the device specifications.
+ * To adjust this function consider change the result variable to desired minimum and maximum value
+ * @param bottomInsets Bottom insets value of the device
+ * @returns Starting snap value in %
+ */
+const getStartingSnapValue = (bottomInsets: number) => {
+  // We set the maximum bottom insets as 34 which is the currently maximum for a device
+  const weight0 = 1 - bottomInsets / 34; // Weight for 13
+  const weight34 = 1 - weight0; // Weight for 16
+
+  // Use the weights to calculate a weighted average
+  const result = weight0 * 13 + weight34 * 16;
+
+  // Round the result
+  return Math.round(result);
+};
```

Comment:
question: Nice function! Two questions, just for my own knowledge:
1) Where does the value of 34 come from? I tried doing a quick search through iOS docs and couldn't find anything.
2) To make sure I'm understanding properly: if the bottom inset is smaller, we want the site list to open at a lower percentage (13% if there is no bottom inset), and a higher percentage if the bottom inset is higher (16% max). This is just because the bottom inset hides more of the bottom list if the bottom inset is higher, right?

Reply:
1. The maximum number is not written anywhere as there can be a device that comes out later which might have an higher value. We set it as 34 because based on our experience (For the top inset is totally different as Androids differ way more)
2. If the bottom inset is near or equal 0 we set it for 13, if the value is near or equal to 34 we set it to 16, the mid values are calculated too, for example an 26 bottom inset which is normal for ios devices will point up to 15. So for an device really small like an iPhone se that has no bottom bar we set 13 as it's the value which looks better.

Feel free to change the minimum and maximum value on the result variable by changing the 13 and 16 to the desired values.